### PR TITLE
Improve consistency/spelling in some comments

### PR DESCRIPTION
**Pull request by salutis**

### CODE REVIEW

File: src/nnn.c [hunk 1]
```diff
@@ -5960,7 +5960,7 @@ static bool browse(char *ipath, const char *session, int pkey)
 	xcols = COLS;
 
 #ifndef NOSSN
-	/* setup first context */
+	/* set up first context */
```
Comment (jarun) [inline src/nnn.c:5963]: Set-up

**Review by jarun — CHANGES_REQUESTED** [review]

File: src/nnn.c (hunk 1 above)
Comment (salutis) [reply src/nnn.c]: Fixed.

File: src/nnn.c [hunk 2]
```diff
@@ -169,7 +169,7 @@
 #define SELECT ' '
 #define REGEX_MAX 48
 #define ENTRY_INCR 64 /* Number of dir 'entry' structures to allocate per shot */
-#define NAMEBUF_INCR 0x800 /* 64 dir entries at once, avg. 32 chars per filename = 64*32B = 2KB */
+#define NAMEBUF_INCR 0x800 /* 64 dir entries at once, avg. 32 chars per file name = 64*32B = 2KB */
```
Comment (jarun) [inline src/nnn.c:172]: Both filename and file name are fine. Can you change back these to `filename`?

**Comment by jarun** [comment]
Thank you!

**Pull request merged**